docs/site/performance.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Performance Tuning
+nav_order: 6
+---
+# Performance Tuning
+
+* TOC
+{:toc}
```

Comment:
What does this do?   Other pages don't seem to have it.

Reply:
It's Table of Contents. FAQ page has this TOC, https://github.com/NVIDIA/spark-rapids-ml/blob/branch-23.10/docs/site/FAQ.md